api-tests/transaction-service/npg/transaction-service_ecommerce_local.postman_environment.json

```diff
@@ -73,6 +73,12 @@
 			"value": "",
 			"type": "default",
 			"enabled": true
+		},
+		{
+			"key": "CORRELATION_ID",
+			"value": "",
+			"type": "default",
+			"enabled": true
```

Comment:
```suggestion
```
This variable can be removed as is only a dynamic variable automatically set by get session test script